block/manager.go

```diff
@@ -78,6 +88,11 @@ func NewManager(
 		return nil, err
 	}
 
+	if conf.DABlockTime == 0 {
+		logger.Info("WARNING: using default DA block time", "DABlockTime", defaultDABlockTime)
+		conf.DABlockTime = defaultDABlockTime
+	}
```

Comment:
Ugh, too bad we're not using Rust here

Reply:
Not to mention how limited is the condition variable in Go...